src/Hyperledger.Aries.Routing.Mediator/Hyperledger.Aries.Routing.Mediator.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
+        <TargetFramework>netcoreapp3.1</TargetFramework>

```

Comment:
This is a packable project, should target netstandard.

Reply:
fluent validation 9.x.x does not work with netstandard anymore so you can not upgrade to ver 9.x.x.

Reply:
Hyperledger.Aries.AspNetCore removed from Hyperledger.Aries.Routing.Mediator and used Microsoft.AspNetCore.Http.Abstractions instead.